python/nammu/view/AtfAreaView.py

```diff
@@ -402,8 +403,36 @@ def undoableEditHappened(self, event):
             # to false. Note undo() only undoes compound edits when they
             # are not in progress.
             self.current_compound.end()
-            self.current_compound = CompoundEdit()
+            self.current_compound = AtfCompoundEdit()
             self.undo_manager.addEdit(self.current_compound)
 
         # Always add current edit to current compound
         self.current_compound.addEdit(edit)
+
+
+class AtfCompoundEdit(CompoundEdit):
+    # add a getter for a protected field
```

Comment:
It's probably worth adding a comment/docstring on this class and `AtfUndoManager` explaining why we're subclassing `CompoundEdit`/`UndoManager` (essentially that we want to expose/access some private fields, and that Jython is weird about protected methods).

It otherwise looks a bit random, and the Jython super-secret syntax is a little opaque.